controllers/grid/queries/form/QueryForm.php

```diff
@@ -392,7 +392,7 @@ public function fetch($request, $template = null, $display = false, $actionArgs
         $allowedEditTimeNotice = ['show' => false, 'limit' => 60];
         if (array_intersect($assignedRoles, [Role::ROLE_ID_ASSISTANT, Role::ROLE_ID_AUTHOR, Role::ROLE_ID_REVIEWER])) {
             $allowedEditTimeNotice['show'] = true;
-            $allowedEditTimeNotice['limit'] = (int) ($allowedEditTimeNotice['limit'] - (time() - strtotime($headNote->getDateCreated())) / 60);
+            $allowedEditTimeNotice['limit'] = (int) ($allowedEditTimeNotice['limit'] - (time() - strtotime($headNote->dateCreated)) / 60);
```

Comment:
If possible, better to use the `Carbon` package to make it more understandable :)